aip/general/0162.md

```diff
@@ -293,7 +204,7 @@ back to a given revision. APIs **should** handle this with a `Rollback` custom
 method:
 
 ```proto
-rpc RollbackBook(RollbackBookRequest) returns (Book) {
+rpc RollbackBook(RollbackBookRequest) returns (BookRevision) {
```

Comment:
I'm not *sure* whether "rollback" is the right terminology to use now that the resources are separate.
Revert and restore seem just as suitable... hmm. (This may have been discussed internally already.)

Reply:
+1 to restore. I think semantically this would make sense to, if there's data-plane information that's attached to a collection as well.